app/routes/events/view/export.js

```diff
@@ -1,9 +1,20 @@
 import Ember from 'ember';
 
-const { Route } = Ember;
+const { Route, RSVP, Object } = Ember;
 
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Export');
+  },
+  model() {
```

Comment:
Entire model definition is not needed here. the event data from the parent route `events.view` will be automatically accessible.